addons/pos_self_order/__manifest__.py

```diff
@@ -23,6 +23,7 @@
     ],
     "demo": [
         "data/kiosk_demo_data.xml",
+        "data/presets_data.xml",
```

Comment:
```suggestion
        "data/presets_data.xml",
        "data/kiosk_demo_data.xml",
```

load before `kiosk_demo_data.xml`